src/openforms/emails/digest.py

```diff
@@ -179,3 +257,80 @@ def collect_broken_configurations() -> list[BrokenConfiguration]:
         )
 
     return broken_configurations
+
+
+def collect_invalid_registration_backends() -> list[InvalidRegistrationBackend]:
```

Comment:
Maybe another implementation could be to split the detection of invalid registration backend instances (where we do `plugin.configuration_options(data=options)`) and the config checks (where we do `plugin.check_config()`).

This would avoid:
- The duplicate check logic you implemented with `already_detected`
- Having everything in the single `InvalidRegistrationBackend` class (the `form_level` boolean attribute makes things confusing, and `form_name/id` is not relevant for the plugin check).

So I think `collect_invalid_registration_backends` could only check for invalid instances (with the `configuration_options` check), and another collect function can be defined to iterate over the registration plugins (using `iter_enabled_plugins`) return a list of a separate dataclass representing invalid plugins.

Reply:
checking a plugin by itself doesn't make sense if the plugin is not actually used in any forms, so looping over `iter_enabled_plugins` will just complicate stuff since you need to look up in which forms it's used and that's a lot harder than starting from the registration backends in the first place.

I think starting from the forms itself is the best approach here to not report on things that don't cause problems and ensure that receivers of the email digest know which form(s) are affected by this problem.